REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.5
-OpenCL
+OpenCL 0.5.2
```

Comment:
I haven't tested it and I'm running out of time ...
Feel free to test and make a PR with the correct version!